### ISSUE TITLE
[Accessibility] Color Contrast ratio in case of Links inside Office fabric dialog should be atleast 4.5:1

### ISSUE THREAD

**Issue by vashitaArora** [OfficeDev/office-ui-fabric-react#1037]
### Bug Report

when using Office Fabric dialog and office fabric links over it, color contrast ratio does not match standard : 4.5:1

#### Actual behavior:

Color Contrast ratio <  4.5:1

#### Expected behavior:

Color Contrast ratio in case of Links inside Office fabric dialog should be atleast 4.5:1

**Comment by dzearing** [comment]
@vashitaArora you need to provide a screenshot of what you're talking about. This is really unclease.

**Comment by dzearing** [comment]
Closing due to no traction.
